src/cli/commands/login.command.ts

```diff
@@ -162,4 +208,44 @@ export class LoginCommand {
             return Response.error(e);
         }
     }
+
+    private async getSsoCode(codeChallenge: string, state: string): Promise<string> {
+        return new Promise((resolve, reject) => {
+            const callbackServer = http.createServer((req, res) => {
+                const urlString = 'http://localhost' + req.url;
+                const url = new URL(urlString);
+                const code = url.searchParams.get('code');
+                const receivedState = url.searchParams.get('state');
+                res.setHeader('Content-Type', 'text/html');
+                if (code != null && receivedState != null && receivedState === state) {
+                    res.writeHead(200);
+                    res.end('<html><head><title>Success | Bitwarden CLI</title></head><body>' +
```

Comment:
Will want to consider localization from language resources or a localized html template/file for this.

Reply:
Probably, however, the CLI does not localize anything today.